library/core/src/fmt/num.rs

```diff
@@ -478,93 +486,6 @@ mod imp {
 }
 impl_Exp!(i128, u128 as u128 via to_u128 named exp_u128);
 
-/// Helper function for writing a u64 into `buf` going from last to first, with `curr`.
-fn parse_u64_into<const N: usize>(mut n: u64, buf: &mut [MaybeUninit<u8>; N], curr: &mut isize) {
```

Comment:
This function was effectively a duplicate of fmt_u64 and seemed slower in a quick benchmark I ran, so I de-duplicated and replaced it with `fmt_u64`.